src/aws/lambda-mediapackage/src/utils/recordSlicesState/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Get the record slices states for a video, using hmac with a shared secret
+ * to sign the object key and provide authorization for the request.
+ */
+const { computeSignature, sendRequest } = require('update-state/utils');
+
+const { RECORDING_SLICES_STATE_ENDPOINT, SHARED_SECRET } = process.env;
+const DISABLE_SSL_VALIDATION = JSON.parse(process.env.DISABLE_SSL_VALIDATION);
+
+module.exports = async (key = {}) => {
```

Comment:
`key` is not relevant here and I don't understand why there is a default value. Why not using `video_id` ? It will be only used in a `video` context